javadoc.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+echo "Javadoc-testing commit $TRAVIS_COMMIT"
+if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ -n "${GITHUB_TOKEN:-}" ]; then
+      curl -H "Content-Type: application/json" -H "Authorization: token $GITHUB_TOKEN" \
+            --request POST \
+            --data '{"state":"pending",
+            "target_url":"https://travis-ci.org/Minecolonies/minecolonies/builds/{$TRAVIS_BUILD_ID}",
+            "description":"Javadoc build in progress...",
+            "context":"javadoc"}' \
+            "https://api.github.com/repos/Minecolonies/minecolonies/statuses/$TRAVIS_COMMIT"
+            
+      echo "Start javadoc gradle..."
+      
+      if ./gradlew javadoc; then
```

Comment:
So, this allows us to create a custom PR Acceptance criteria that we change using api calls?